client/src/components/HistoryViewer/HistoryViewerVersionList.scss

```diff
@@ -0,0 +1,91 @@
+// Replicate Bootstrap table stylings relatively closely but using a unordered list
+.history-viewer__table {
+  padding: 0;
+
+  li {
+    display: flex;
+    list-style-type: none;
+  }
+}
+
+// In compare mode, we don't want any space between the two version lists, only a small black line.
+.history-viewer__table--comparison-selected {
+  margin-bottom: 0;
+  border-bottom: 5px solid $gray-800;
+
+}
+
+.history-viewer__row--comparison-selected:last-child {
+  border-bottom: none;
+}
+
+.history-viewer__heading,
+.history-viewer__row {
+  border-bottom: 1px solid $table-border-color;
+}
+
+// Add colours to the table rows
+.history-viewer__row {
+  background-color: $table-accent-bg;
+
+  &:hover,
+  &:active {
+    background-color: $table-hover-bg;
+  }
+}
+
+.history-viewer__version-anchor {
+  color: inherit;
+  display: flex;
+  flex-grow: 1; // use up 3 columns worth of space by default
+
+  &:hover,
+  &:active {
+    color: inherit;
+    text-decoration: none;
+  }
+}
+
+// Give all columns equal width to start with, and table padding
+.history-viewer__version-no,
+.history-viewer__version-state,
+.history-viewer__author,
+.history-viewer__actions {
+  padding: $table-cell-padding;
+}
+
+.history-viewer__version-state,
+.history-viewer__author {
+  flex-grow: 1;
+  width: 50%;
+}
+
+// Remove padding in the heading's last column so it doesn't push padding in the others
+.history-viewer__actions {
+  padding-top: 0;
+  padding-bottom: 0;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end; // right alignment
+  min-width: 12rem; // big enough to fit both Compare and "X" close buttons
+}
+
+.table .history-viewer__row--current {
+  &, &:hover {
+    background-color: $link-color;
+    color: $white;
+  }
+
+  .text-muted {
+    color: $white !important;
```

Comment:
I understand the root cause of this problem is that `admin` uses `!important` for `.text-muted`. But we're compounding the problem.

Do a separate PR to fix admin.

Reply:
It actually comes from Bootstrap